config/environments/development.rb

```diff
@@ -42,6 +42,8 @@
   config.auth0_client_id = ENV['MUMUKI_AUTH0_CLIENT_ID']
   config.auth0_client_secret = ENV['MUMUKI_AUTH0_CLIENT_SECRET']
   config.auth0_domain = ENV['MUMUKI_AUTH0_DOMAIN']
+  config.auth_provider = ENV['MUMUKI_AUTHORIZATION_PROVIDER'] # auth0 or saml (case insensitive)
```

Comment:
Please don't add comments to the code. 

If you think this information is meaningful - I think *YES* -, add it to the README.